components/search/SearchForm.vue

```diff
@@ -3,32 +3,54 @@
     inline
     @submit.prevent="submitForm"
   >
-    <b-form-input
-      v-model="query"
-      :aria-label="$t('search')"
-      :placeholder="$t('searchPlaceholder')"
-      name="query"
-      data-qa="search box"
-      @input="$emit('input', $event)"
-    />
-    <b-button
-      type="submit"
-      data-qa="search button"
-      variant="primary"
-    >
-      <span>
-        {{ $t('search') }}
-      </span>
-      <img
-        src="../../assets/img/magnifier.svg"
-        :alt="$t('search')"
+    <b-input-group>
+      <template
+        v-if="pillLabel"
+        v-slot:prepend
       >
-    </b-button>
+        <SearchBarPill
+          :text="pillLabel"
+          :remove-link-label="$t('removeEntityFilter', { entityLabel: pillLabel })"
+          :remove-link-to="{
+            path: localePath({
```

Comment:
instead of having multiline code in the template here, should this be a computed property or method?

Reply:
Agreed, let's do this already.